<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_Main_Form</class>
 <widget class="QWidget" name="UI_Main_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>620</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>一、二维码生成软件</string>
  </property>
  <widget class="Common_Button" name="btnImport">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>9</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择新的背景</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>9</y>
     <width>481</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>600</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>生成二维码</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>22</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="Common_Button" name="btncre2D">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>生成二维码</string>
        </property>
        <property name="shortcut">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Common_Button" name="save2D">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>保存为图片</string>
        </property>
        <property name="shortcut">
         <string>H</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>420</width>
        <height>390</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>390</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>生成条形码</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEdit1D">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="Common_Button" name="btncre1D">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>生成条形码</string>
        </property>
        <property name="shortcut">
         <string>H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Common_Button" name="save1D">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string>保存为图片</string>
        </property>
        <property name="shortcut">
         <string>H</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Common_Button" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>关于本软件…</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Common_Button</class>
   <extends>QPushButton</extends>
   <header>state_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
